src/api/test/functional/maintenance_test.rb

```diff
@@ -1743,6 +1743,11 @@ def test_create_maintenance_project_and_release_packages
     assert_xml_tag tag: 'directory', attributes: { vrev: '42.1' } # X.Y scheme
     delete '/source/BaseDistro2.0:ServicePack1/packNEW'
     assert_response :success
+    # clean up
+    delete '/source/BaseDistro2.0:LinkedUpdateProject/packNEW'
```

Comment:
The check below is not needed, it is making the test slower, if the delete fails I guess we're going to find out on the rest of the test

Reply:
You mean the second `get` and the `assert_response :not_found`, right?

Although I see the redundancy, it's been a bit of a PITA to figure out what was the issue with the unrelated failing test caused by this afterwards, I wanted to make it sure the scenario is clean as we expect it is.

Reply:
Okay, but once it is fixed, that won't change, right? I think there is no need to check for success or failure if neither the test or the conditions change from run to run...